public/docs/ts/latest/cookbook/production-deploy.jade

```diff
@@ -0,0 +1,327 @@
+include ../_util-fns
+  
+:marked
+  This cookbook describes how to optimize and deploy your Angular app to production.
+
+a#toc
+:marked
+  ## Table of Contents
+  * [Overview](#overview)
+  * [Simplest deploy possible](#simplest-deploy-possible)
+  * [Minimizing your payload](#minimizing-your-payload)
+    * [Fewer files](#fewer-files)
+    * [Smaller files](#smaller-files)
+    * [Template compilation](#template-compilation)
+  * [Angular configuration](#angular-configuration)
+    * [The `base` tag](#the-base-tag)
+    * [enableProdMode](#enableprodmode)
+    * [Lazy loading](#lazy-loading)
+  * [Server configuration](#server-configuration)
+    * [Why fallback to `index.html`?](#why-fallback-to-index-html-)  
+    * [Fallback configuration examples](#fallback-configuration-examples)  
+
+a#overview
+.l-main-section
+:marked
+  ## Overview
+
+  After creating your brand new Angular app you'll want to put it online for the world to see.
+
+  Your development setup is optimized for build speed and rapid iteration, but when
+  deploying to production you'll want to optimize for loading speed and payload size.
+
+  In this chapter you'll see how to deploy your app right now, techniques to reduce your 
+  payload size, plus Angular and server configuration.
+
+.l-main-section
+:marked
+  ## Simplest deploy possible
+
+  The simplest possible way to deploy your app is to use your development environment.
+  It is already working locally after all, so it should work live.
+
+  1. Copy over your local project folder to your server.
+  2. On your server, edit `index.html` to have the correct `<base href="/">` tag. If you are
+  using a subfolder (e.g. `www.mysite.com/myapp/`) it should be `/myapp/` otherwise leave it as `/`.
+  [More on this later](#angular-configuration).
+  3. Configure your server to redirect requests for missing files to `index.html` instead.
+  [More on this later](#server-configuration).
+
+  And this is all you need to publish your app!
+
+  It's not a very good production deploy though. 
+
+  For one, it has a lot of files that aren't even used when serving your site to users since it has
+  all the development only packages. 
+  You could cut down the number of files by doing a production install (`npm install --production`),
+  but only *after* compiling your Typescript files.
+
+  It also has a lot of small files. Each of those is going to take a whole round trip to get
+  delivered to the browser, so it's not going to be that fast to load.
+
+  There's more you can do to make our site fast for users.
+ 
+.l-main-section
+:marked
+  ## Minimizing your payload
+
+  One of the easiest optimizations you can do is to reduce the code users have to download.
+  
+  There are several techniques for achieving a smaller payload for your app:
```

Comment:
Great list! We all know what these do, but I think we should add a sentence here that explains succinctly the net effect of these tasks. Perhaps add ... `which will reduce the load time and improve the user experience.`